docs/_static/site.css

```diff
@@ -0,0 +1,39 @@
+
+.iver-logo-wordmark, .iver-logo-symbol {
+	line-height: 6rem;
+	display: block;
+}
+
+.iver-logo-wordmark svg .st0, .iver-logo-symbol svg .st0 {
+	fill: #000000;
+}
+
+.iver-logo-wordmark {
+	float: left;
+}
+.iver-logo-wordmark svg {
+	/* logotype wordmark shall be 90% height of the symbols height */
+	height: 2.7rem;
+}
+
+.iver-logo-symbol {
+	float: right;
+}
+.iver-logo-symbol svg {
+	height: 3rem;
+}
+
+/* Utility classes 
+ *
+ * Example usage:
+ * ![my image](_images/some-image.png ':class=float-right')
+ *
+ * */
```

Comment:
Remove? :)

Reply:
Why?

Reply:
I skimmed it too fast. I thought it was just commented out CSS.

I think those classes are self explanatory imo though.

Reply:
Well the usage is not self-explanatory, but that kind of documentation shouldn't live there anyway.

I ended up removing it as it's unused anyway